coalib/bearlib/languages/__init__.py

```diff
@@ -3,40 +3,56 @@
 """
 
 # Start ignoring PyUnusedCodeBear
-from .Language import Language
-from .Language import Languages
-
-from .definitions.Unknown import Unknown
 from .definitions.Bash import Bash
+
 from .definitions.C import C
 from .definitions.CPP import CPP
-from .definitions.CSharp import CSharp
 from .definitions.CSS import CSS
+from .definitions.CSharp import CSharp
+
 from .definitions.Fortran import Fortran
+
 from .definitions.Golang import Golang
+
 from .definitions.html import HTML
-from .definitions.Java import Java
-from .definitions.JavaScript import JavaScript
+
 from .definitions.JSON import JSON
 from .definitions.JSP import JSP
+from .definitions.Java import Java
+from .definitions.JavaScript import JavaScript
+from .definitions.Jinja2 import Jinja2
+
 from .definitions.KornShell import KornShell
-from .definitions.m4 import m4
-from .definitions.Matlab import Matlab
+
+from .Language import Language
+from .Language import Languages
+
 from .definitions.Markdown import Markdown
+from .definitions.Matlab import Matlab
+from .definitions.m4 import m4
+
 from .definitions.ObjectiveC import ObjectiveC
+
 from .definitions.PHP import PHP
 from .definitions.PLSQL import PLSQL
 from .definitions.PowerShell import PowerShell
 from .definitions.Python import Python
+
 from .definitions.Ruby import Ruby
+
 from .definitions.Scala import Scala
+from .definitions.Shell import Shell
 from .definitions.Swift import Swift
+
 from .definitions.Tcl import Tcl
-from .definitions.Vala import Vala
 from .definitions.TypeScript import TypeScript
-from .definitions.Shell import Shell
-from .definitions.Jinja2 import Jinja2
+
+from .definitions.Unknown import Unknown
+
+from .definitions.Vala import Vala
 from .definitions.VisualBasic import VisualBasic
+
 from .definitions.XML import XML
+
```

Comment:
I see you are adding a lot of blank lines. IMO these are not required, please only arrange imports in alphabetic order, don't add/remove lines unnecessarily.

Reply:
okay. Thank you for the review @pareksha  @mach64

Reply:
i'll make the necessary changes as suggested

Reply:
As suggested by others remove unnecessary blank spaces from the code